src/pools/abstract-pool.ts

```diff
@@ -87,6 +87,10 @@ export interface PoolOptions<Worker> {
    * The work choice strategy to use in this pool.
    */
   workerChoiceStrategy?: WorkerChoiceStrategy
+  /**
+   * Pool events emitting.
+   */
+  emitEvents?: boolean
```

Comment:
From the name of this variable, it's not clear to me what it's doing.  
Can you try to find a better name?